File: ScanbotSDKDemo/BusinessCardDemo/BusinessCardDemoCell.m
//
//  BusinessCardDemoCell.m
//  SBSDK Internal Demo
//
//  Created by Yevgeniy Knizhnik on 9/26/18.
//  Copyright © 2018 doo GmbH. All rights reserved.
//

#import "BusinessCardDemoCell.h"

@interface BusinessCardDemoCell ()
@property (strong, nonatomic) IBOutlet UIImageView *imageView;

@end

@implementation BusinessCardDemoCell

-(void)awakeFromNib {
    [super awakeFromNib];
    self.imageView.image = self.image;
}

- (void)setImage:(UIImage *)image {
    _image = image;
    self.imageView.image = image;
}

@end

File: ScanbotSDKDemo/BusinessCardDemo/BusinessCardDemoCell.swift
//
//  BusinessCardDemoCell.swift
//  ScanbotSDKDemo
//
//  Created by Danil Voitenko on 26.11.21.
//  Copyright © 2021 doo GmbH. All rights reserved.
//

import UIKit

class BusinessCardDemoCell: UICollectionViewCell {
    var image: UIImage? {
        didSet {
            imageView?.image = image
        }
    }
    
    @IBOutlet private var imageView: UIImageView?
    
    override func awakeFromNib() {
        super.awakeFromNib()
        
        imageView?.image = image
    }
}
